static/app/views/replays/list/replayPanel.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled';
+
+import Panel from 'sentry/components/panels/panel';
+import {space} from 'sentry/styles/space';
+
+interface Props extends React.ComponentProps<typeof Panel> {
+  children: React.ReactNode;
+  image?: React.ReactNode;
+  noCenter?: boolean;
+}
+
+function ReplayPanel({image, noCenter, children, ...props}: Props) {
+  return (
+    <Panel {...props}>
+      <Container>
+        {image ? <IlloBox>{image}</IlloBox> : null}
+        <StyledBox centered={!image && !noCenter}>{children}</StyledBox>
+      </Container>
+    </Panel>
+  );
+}
+
+const Container = styled('div')`
+  padding: ${space(3)};
+  position: relative;
+
+  @media (min-width: ${p => p.theme.breakpoints.small}) {
+    display: flex;
+    align-items: flex-start;
```

Comment:
Basically main CSS change here is that items are aligned flex-start so that the image doesn't shift around when the onboarding box expands/collapses.